tethysapp/streamflow_prediction_tool/public/css/map.css

```diff
@@ -1,15 +1,16 @@
 /* MAP STYLES */
 #app-content-wrapper #app-content {
     height: 100%;
+    padding-bottom: 0;
```

Comment:
I like the padding, but will leave everything else the same. Would you mind adding this to a separate PR?